src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -716,14 +732,15 @@ def __init__(self, config):
         super().__init__()
 
         self.use_batch_norm = config.use_batch_norm_in_fusion_residual
+
         self.activation1 = ACT2FN["relu"]
         self.convolution1 = nn.Conv2d(
             config.fusion_hidden_size,
             config.fusion_hidden_size,
             kernel_size=3,
             stride=1,
             padding=1,
-            bias=not self.use_batch_norm,
```

Comment:
This isn't backwards compatible, users might have custom configs with non-default setting for `use_batch_norm_in_fusion_residual`. We need to do something along the lines of the following: 

* Default value of `use_bias_in_fusion_residual` and `use_batch_norm_in_fusion_residual` are `None` in the config
* If `use_batch_norm_in_fusion_residual` is set, then `use_bias_in_fusion_residual` takes this value, we emit a warning that this will change in the future and they should use `use_bias_in_fusion_residual`
* If both `use_batch_norm_in_fusion_residual` and `use_bias_in_fusion_residual` are set, then we use `use_bias_in_fusion_residual` and emit a warning that `use_batch_norm_in_fusion_residual` is being ignored
* If neither are set then `use_bias_in_fusion_residual` takes the default value of `True`

Reply:
Agree on keeping backwards compatibility, however I think this will throw warnings for each case, which is something we might need to avoid (I'm in favor of trying to avoid warnings as much as possible for user experience).

With the approach above, we cause the more popular [DPT-midas](https://huggingface.co/Intel/dpt-hybrid-midas) (> 2 million downloads) and [DPT-large](https://huggingface.co/Intel/dpt-large) (nearly 300k downloads) to always have a warning, whereas this batch norm was only used for [DPT-ade](https://huggingface.co/Intel/dpt-large-ade) (which almost no one is using - it has 800 downloads).

Hence I've pushed a change which tries to avoid warnings, let me know what you think